widgets/htdocs/components/95_AFDB.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2017] EMBL-European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ensembl.Panel.AFDB = Ensembl.Panel.Content.extend({
+
+  init: function() {
+    this.base.apply(this, arguments);
+
+    if (this.areStaticImportsSupported()) {
+      this.loadScript();
+    } else {
+      this.onUnsupportedBrowser();
+    }
+  },
+
+  loadScript: function () {
+    this.addSpinner();
+
+    var script = document.createElement('script');
+    script.setAttribute('src', '/alphafold/index.js'); // <-- all the fun is happening here
+    script.setAttribute('type', 'module');
+    script.onload = this.onScriptLoaded.bind(this);
+    script.onerror = this.onScriptLoadError.bind(this);
+    document.body.appendChild(script);
+  },
+
+  onScriptLoaded: function() {
+    var enspIdElement = document.querySelector('#ensp_id'); // <-- expecting 1 or 0 HTML input elements
+    if (!enspIdElement) {
+      this.onMissingData();
+      return;
+    }
+
+    var container = document.querySelector('#alphafold_container');
+    var ensemblAlphafoldElement = document.createElement('ensembl-alphafold-viewer');
```

Comment:
I am just thinking instead of creating the element here, you can create it already in the perl file and instead of using the extra setAttribute to set the data attribute, you can set it already in the perl side. This way it keeps the code cleaner and also avoid an extra layer of converting the data (perl -> jquery -> web-component). And in the future if you need an extra parameter, you can just add it from the perl module. Does it make sense?

Reply:
Yes, fair point. I have moved the code for setting up the web component to the server.